packages/api/src/apps/serverless_read/app.ts

```diff
@@ -118,8 +118,8 @@ export default async (env: ProcessEnv) => {
       projectName: env.TENDERLY_PROJECT_NAME,
       accessKey: env.TENDERLY_ACCESS_KEY,
     };
-    const tenderlies = new libs.osnap.MultiChainTenderly(tenderlyConfig);
-    channels.push(["osnap", Actions.Osnap({ tenderlies })]);
+    const tenderly = new libs.osnap.utils.TenderlyApi(tenderlyConfig);
```

Comment:
changing approach here. turns out the typescript lib they provide differs from the restful api. we need the simulation ID which only the restful tenderly api returns, so now we use that instead